tests/rabbitmq_handlers_test.go

```diff
@@ -236,13 +236,13 @@ func TestEstimateHandler_InvalidDuration(t *testing.T) {
 	assert.Equal(t, errors.New("error with the parsed estimation object: \n"+expectedErr.Error()), err)
 }
 
-func TestEstimateHandler_InvalidUUID(t *testing.T) {
+func TestEstimateHandler_EmptyUUID(t *testing.T) {
 	//Arrange
 	mockPublisher := new(mocks.PublisherInterface)
 
 	handler := handlers.EstimateHandler{Publisher: mockPublisher}
 
-	estimate := models.Estimation{Origin: "Weston, Fl", Destination: "Miami, Fl", UserId: "0", Duration: 3000, Distance: 3000}
+	estimate := models.Estimation{Origin: "Weston, Fl", Destination: "Miami, Fl", UserId: "", Duration: 3000, Distance: 3000}
```

Comment:
@jacobkissel how come you switched it from invalid UUID to empty? Would "0" as a UUID work?

Reply:
We no longer wanted to verify UUID's so I'm just verifying that one is there. So technically yeah 0 would work lol. The UUID verification was messing up because of the issues with the UUIDs in the db.